test/tasks/vesting.test.ts

```diff
@@ -35,16 +35,24 @@ const MOCK_VESTING_HEADER = [
   'lastQuantityRevoked',
 ];
 
+const currentDate = new Date();
+const dateLessThanOneYearAgoFromToday = new Date(
+  currentDate.getFullYear() - 1,
+  currentDate.getMonth(),
+  currentDate.getDate() + 1
+);
+const currentDatePlusOne = new Date(currentDate.getTime() + 1);
+
 const MOCK_GITHUB_VESTING_GRANTS = [
   [
     '0xDD66B46910918B2F442D6b75C6E55631ad678c99',
     '4c728e75-0aef-42f6-8ee3-3a831734e048',
     '46425.5886',
-    '2023-07-14T00:00:00Z',
+    dateLessThanOneYearAgoFromToday.toISOString(),
```

Comment:
could probably do toISOString once when initializing the value so we dont need it in each test